cmd/run.go

```diff
@@ -85,6 +83,11 @@ func (r *RunConfig) flags() []cli.Flag {
 			Usage:       "Do not delete the temp directory after the run",
 			Destination: &r.RetainTempDir,
 		},
+		&cli.StringFlag{
+			Name:        "prepared-dir",
+			Usage:       "Relative directory already prepared. Cannot include version with this.",
```

Comment:
Could add a comment that in the `version` help string that it's mutually exclusive with `prepared-dir` too.

Reply:
I think we're filling up these CLI arg docs with too much info. Note how I also don't tell the format of version for each language too. But I'll do it. However, keep in mind we need to be keeping these brief.